contrib/systemd/cilium

```diff
@@ -1,3 +1,3 @@
 PATH=/usr/lib/llvm-3.8/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/sbin:/sbin:/bin
-CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500
+CILIUM_OPTS=--kvstore consul --kvstore-opt consul.address=127.0.0.1:8500 --debug
```

Comment:
Will this affect all deployments, or just in the testing? Not sure if this is unrelated change that should be kept out.

Reply:
My fault! I'll remove it ;-)